concept/type/attributetype.feature

```diff
@@ -11,83 +11,76 @@ Feature: Concept Attribute Type
     Given connection has been opened
     Given connection does not have any database
     Given connection create database: typedb
-    Given connection opens schema transaction for database: typedb
+    Given connection open schema transaction for database: typedb
+
+    # TODO: Create structs in concept api
```

Comment:
Every time I create a struct I leave this TODO marking that we might need to refactor these steps after we add 'final' tests for structs. 
We may decide the Concept API for structs here, so I could change these lines and remove the TODOs, if we want to.